test/unit/foreman/renderer/scope/macros/base_test.rb

```diff
@@ -96,12 +96,15 @@ class BaseMacrosTest < ActiveSupport::TestCase
       chef_kernel_fact = FactoryBot.create(:fact_name, name: 'kernel::release', :type => 'FactName::Chef')
       unrelated_fact = FactoryBot.create(:fact_name, name: 'os')
       puppet_and_salt_fact = FactoryBot.create(:fact_name, name: 'kernelrelease')
+      rhsm_fact = FactoryBot.create(:fact_name, name: 'uname::release', :type => 'Katello::RhsmFactName')
       FactoryBot.create(:fact_value, fact_name: ansible_kernel_fact, host: host, :value => '1.2.3')
       FactoryBot.create(:fact_value, fact_name: chef_kernel_fact, host: host, :value => '2.2.2')
       FactoryBot.create(:fact_value, fact_name: unrelated_fact, host: host, :value => 'Fedora 29')
       assert_equal '1.2.3', @scope.host_kernel_release(host)
       FactoryBot.create(:fact_value, fact_name: puppet_and_salt_fact, host: host, :value => '4.5.6')
       assert_equal '4.5.6', @scope.host_kernel_release(host.reload)
+      FactoryBot.create(:fact_value, fact_name: rhsm_fact, host: host, :value => '7.8.9')
+      assert_equal '7.8.9', @scope.host_kernel_release(host.reload)
```

Comment:
Will this work? It's last in the list and the other facts already exist so I wouldn't expect it to be returned. Others have a higher priority.

Reply:
Thanks, @ekohl  for the help. I do not have a test environment configure so I updated the test case without running the test at local.
I hope the test will be green now